features/docs/defining_steps/ambiguous_steps.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Ambiguous Steps
```

Comment:
If you tag this feature as `@wip` then (I think) it will be expected to fail in CI, so your branch will stay green. Once we have it passing we can remove the `@wip` tag.